projects/1.19/assets/chat-patches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,60 @@
+{
+    "text.chatpatches.title": "聊天补丁设置",
+
+    "text.chatpatches.dateFormat": "要设置日期和时间格式化样式，点我！",
+    "text.chatpatches.formatCodes": "要设置字符格式化代码和行为，点我！",
```

```suggestion
    "text.chatpatches.formatCodes": "要设置字符格式化代码和获取相关帮助，点我！",
```
原文应该有提`which one does which`